src/api/status/public/index.html

```diff
@@ -32,6 +32,8 @@
     <link href="assets/css/nucleo-svg.css" rel="stylesheet" />
     <!-- Font Awesome Icons -->
     <script src="https://kit.fontawesome.com/42d5adcbca.js" crossorigin="anonymous"></script>
+    <!-- GitHub Stats -->
+    <script type="text/javascript" src="gitHubStats.js"></script>
```

Comment:
This needs to be moved before the closing `body` tag. The script file should also be inside the js folder.

https://github.com/Seneca-CDOT/telescope/blob/06c71038ad617c0ac805e4f3ccc9cbb1147862c3/src/api/status/public/pages/build.html#L32-L44

Reply:
done